readme.md

```diff
@@ -124,6 +124,7 @@ NOD | Non-Deterministic tests
 NDOD | Non-Deterministic Order-Dependent tests that fail non-deterministically but with significantly different failure rates in different orders as defined in our [ISSRE’20 work](http://mir.cs.illinois.edu/winglam/publications/2020/LamETAL20ISSRE.pdf)
 NDOI | Non-Deterministic Order-Independent tests that fail non-deterministically but similar failure rates in all orders as defined in our [ISSRE’20 work](http://mir.cs.illinois.edu/winglam/publications/2020/LamETAL20ISSRE.pdf)
 UD | Unknown Dependency tests that pass and fail in a test suite or in isolation
+Portability | Tests that pass and fail depending on the operating system
```

Comment:
How about calling them `OSD` for `Operating System Dependent`? Portability may be too broad as it can depend on many aspects of a platform not just OS.

Reply:
Okay, will change that.